app/assets/javascripts/discourse/app/lib/post-bookmark-manager.js

```diff
@@ -0,0 +1,78 @@
+import { setOwner } from "@ember/application";
```

Comment:
We would also have a TopicBookmarkManager and ChatBookmarkManager and insert whatever one makes sense into BookmarkMenu